Project/Assets/ML-Agents/Examples/Soccer/Scripts/AgentSoccer.cs

```diff
@@ -72,21 +104,21 @@ public void MoveAgent(float[] act)
         switch (forwardAxis)
         {
             case 1:
-                dirToGo = transform.forward * 1f;
+                dirToGo = transform.forward * m_ForwardSpeed;
                 m_KickPower = 1f;
                 break;
             case 2:
-                dirToGo = transform.forward * -1f;
+                dirToGo = transform.forward * -m_ForwardSpeed;
                 break;
         }
 
         switch (rightAxis)
         {
             case 1:
-                dirToGo = transform.right * 0.3f;
+                dirToGo = transform.right * m_LateralSpeed;
                 break;
             case 2:
-                dirToGo = transform.right * -0.3f;
+                dirToGo = transform.right * -m_LateralSpeed;
```

Comment:
noticed down below that `Time.deltaTime` is used for rotation but not for `AddForce`.  Have any ideas why?